api/app/src/shared/analytics.ts

```diff
@@ -0,0 +1,32 @@
+import { PostHog } from "posthog-node";
+import { Request } from "express";
+
+import { Config } from "./config";
+import { getCxId } from "../routes/util";
+
+export const queryAnalytics = ({
```

Comment:
I might be missing a bit of context here, but it seems we could rename this function to represent the action we're performing here.